speclang_modern.tex

```diff
@@ -3389,6 +3387,27 @@ \section{Ghost variables and statements}
   \listinginput{1}{ghostcfg.c}
 \end{example}
 
+The qualifier \lstinline|\ghost| can be used, in ghost code only, in declaration
```

Comment:
The purpose of this `\ghost` qualifier is to raise the number of cases where syntactical ghost code can be detected as  being semantically ghost. That point is not enough highlighted.